domains/certificates/Query.API/API/GranularCertificateIssuer/EnergyMeasuredConsumer.cs

```diff
@@ -77,6 +77,11 @@ public async Task Consume(ConsumeContext<EnergyMeasuredIntegrationEvent> context
         if (energyMeasuredIntegrationEvent.Quality != MeasurementQuality.Measured)
             return false;
 
+        if (contract.EndDate != null && energyMeasuredIntegrationEvent.DateTo > contract.EndDate?.ToUnixTimeSeconds())
```

Comment:
I would like that we think methods or variables when making if statements with more than one statement. 
```
if (somethingCoolNameDescribingWhatTheCheckIs())
´´´